backend/backend-schema-proposal.md

```diff
@@ -39,59 +39,123 @@ GET `/request/{van_id}`
 GET `/location`
 
 - Subscribes the client to SSE triggered by new coordinates to any van
-- Responses are in the format `(lat, lon, route_id, van_id, timestamp)` where
-  `van_id` & `route_id` are 8-bit ints, `lat` & `lon` are 64-bit doubles, and
-  `timestamp` is a UNIX timestamp
 - Any response with an earlier timestamp than the most recent should be discarded
+- Excessive time drifting will be detected on backend and reported in admin panel
 - Any timestamp more than 1 min in the future should be discarded
 - Communication can be HTTP or HTTPS, but HTTPS is already being used
+- Response body is defined in JSON as per the following typescript spec:
 
-GET `/routes`
+``typescript
+interface Location {
+    vanId: number,
+    routeId: number,
+    location: Coordinate,
+    timestampMillis: number
+}
+``
 
-- Retrieves a predefined list of routes and their waypoints
+GET `/dashboard`
+
+- Retrieves all data to show on the main screen of the frontend
```

Comment:
something about this seems like bad practice

Reply:
Well, we don't want to underfetch, and I'm not comfortable with calling it `/routes` since it contains information completely unrelated to routes. Any ideas?